lightning/src/lib.rs

```diff
@@ -17,6 +17,29 @@
 //! figure out how best to make networking happen/timers fire/things get written to disk/keys get
 //! generated/etc. This makes it a good candidate for tight integration into an existing wallet
 //! instead of having a rather-separate lightning appendage to a wallet.
+//! 
+//! Available features are: 
+//! 
+//! 1. Skip logging of messages at levels below the given log level:
+//! * `max_level_off` 
+//! * `max_level_error`
+//! * `max_level_warn `
+//! * `max_level_info`
+//! * `max_level_debug`
+//! * `max_level_trace`
+//! 
+//! 2. Allow signing of local transactions that may have been revoked or will be revoked, 
+//! for functional testing (e.g. justice tx handling): 
+//! * `unsafe_revoked_tx_signing` - allows unsafe signing in dev code, such as functional testing of justice transactions outside our crate
+//! * `_bench_unstable` - allows test features for benching
+//! * `no-std ` - exposes write trait implementations from the `core2` crate
+//! 
+//! `default` features are: 
+//! 
+//! * `std` - enables things which require `std`, including `std::io` trait implementations and things which utilize time
```

Comment:
nit: "things" is a bit unspecific/casual here. Maybe "functionalities", avoiding the obvious "features"?